src/shared/components/Accordion.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { Colors } from "@shared/constants";
+import { ReactNode, useState } from "react";
+import { Text } from "@shared/components";
+
+interface AccordionProps {
+    title: string,
+    children: ReactNode;
+}
+
+const Accordion = ({ title, children }: AccordionProps) => {
+    const [isOpen, setIsOpen] = useState(true);
+    const toggleAccordion = () => setIsOpen(!isOpen);
```

Comment:
```suggestion
const toggleAccordion = () => setIsOpen(prev => !prev);
```
이전 상태를 그대로 사용하고자 하면 setState에서 이전 상태를 가져오는 것이 좋습니다!

Reply:
아하 좋네요 감사합니다!!